.gitignore

```diff
@@ -1,5 +1,6 @@
 /node_modules
 /public/hot
+/public/img
```

Comment:
This means we'll have to run our build process on the server, which we usually try to avoid.

Reply:
Gotcha, I'm use to keeping assets out of `public/`.

Reply:
Yah, there are two primary workflows: commit your assets or don't commit them and build on the server. At least right now, Tighten is in the commit your assets camp